scapy/utils.py

```diff
@@ -1657,23 +1669,20 @@ def _read_packet(self, size=MTU):  # type: ignore
 
     def _read_options(self, options):
         # type: (bytes) -> Dict[str, Any]
-        """Section Header Block"""
-        opts = self.default_options.copy()  # type: Dict[str, Any]
+        opts = dict()
```

Comment:
Please explain this change.

Reply:
_read_options() function is called by _read_block_shb(), _read_block_idb() and _read_block_epb()
The option codes may have different meaning for different blocks (eg: code 2 mean if_name for IDB, while for EPB it means epb_flags)
So, i thought it would be better to simplify this function to just read generic codes, and leave the treatment and default option behavior to the callers.